lib/scripts/eventPage.js

```diff
@@ -238,127 +231,114 @@ function contextMenuClickHandler(info, tab) {
     active: true
   }, (tabs) => {
     var tabURL = tabs[0].url;
-    switch (info.menuItemId) {
```

Comment:
What was the motivation to change the switch statement to a series of if statements? The switch statement seemed less verbose.